tools/osbuild-image-info

```diff
@@ -2567,6 +2571,52 @@ def partition_is_lvm(part: Dict) -> bool:
     return part["type"].upper() in ["E6D6D379-F507-44C2-A23C-238F2A3DF928", "8E"]
 
 
+def find_fstab_root(tree, fstype):
+    """
+    Find the root volume under tree by searching for /etc/fstab.
+    This function first checks if the path <tree>/etc/fstab exists and then, if the fstype is btrfs, checks all
```

Comment:
Hm, maybe I'm reading this wrong the code does not quite match the description, i.e. if <tree>/etc/fstab exists it just returns the tree regardless of if it is btrfs or not. 

Maybe ` This function first checks if the path <tree>/etc/fstab exists and if not check all btrfs subvolumes (if any)` (or something)?

Reply:
Modified the docstring.  PTAL.